tools/docker/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
```

Comment:
Add more documentation about this file.
Differences in using host and jetson. 
Where to run it from.